swift/aperture/AudioDeviceList.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  AudioDeviceList.swift
+//  aperture
+//
+//  Created by Khaled Garbaya on 30/11/2016.
+//  Copyright © 2016 Wulkano. All rights reserved.
+//
+
+//: Playground - noun: a place where people can play
+
+import Cocoa
+import CoreAudio
+import Foundation
+import AVFoundation
+import AudioToolbox
+
+class AudioDeviceList {
+
+func getInputDevices() -> [String] {
+    
+    var inputDevices: [String] = []
+    var captureDevices: [AnyObject] = []
+    captureDevices = AVCaptureDevice.devicesWithMediaType(AVMediaTypeAudio)
+    
+    for device in captureDevices {
+        let str: String = "{\"id\": \"\((device.uniqueID as String!))\", \"name\": \"\(device.localizedName)\"}";
```

Comment:
Would be better to properly serialize this into JSON: https://developer.apple.com/swift/blog/?id=37